docker_files/home/lisk/wait-for-it.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
This file is not relevant to lisk docker image, so it should not be included here. Even we can't see any usage of this script file in our Dockerfile.

Its dependency is used by docker-composer under `lisk-docker` repo, so this script should be kept and managed there.

Reply:
I thought on the topic further. If you really want lisk container to wait for db port to be available before running. Then use this script inside `run.sh` not inside the docker-compose. This will make sure that our image will always have this behavior. And then it also makes sense to make this script file part of this repo.

Reply:
That is exactly what I plan on doing, like so:
https://github.com/LiskHQ/lisk-docker/blob/2.1.0/examples/betanet/docker-compose.yml#L15

Reply:
No you are running the script in docker-compose.

Please do it in `run.sh` and in compose start the container. The `run.sh` file should have usage of `wait-for-it.sh` to wait for db to appear online and then run `node app.js`

Reply:
If we were to call `wait-for-it.sh` in `run.sh` then
- it would not be optional any more and
- passing parameters (host and port of the db to wait for) would be akward.

I think it makes sense to keep it where optional and as it is right now (`wait-for-it.sh` waiting for the DB and then calling `run.sh`)

Reply:
If its an optional then don't copy it in container through docker file. 

That's clearly seems a utility script used in docker-compose, then keep it there, mount the utility scripts volume to container and run it. 

If something not required or not related to an image, don't copy it.

Reply:
By optional I mean one can use it but one does not have to.
To give this possibility the script needs to be in the image though (the lisk container must wait for the postgres container; having it somewhere else would not work.

Reply:
> the lisk container must wait for the postgres container
Then make it part of `run.sh`

Having a utility script in image, but not using it somewhere else (composer) does not make sense to me.

Reply:
If we make it part of `run.sh` the it becomes very much non optional. Also it would make passing parameters hard; see above.

> Having a utility script in image, but not using it somewhere else (composer) does not make sense to me.

We have a script in the image, it can be used in one's `docker-compose.yml` or by swarm or rancher or whatever. Or one may choose not to use it. That's why I call it "optional".

Reply:
@fchavant Ok please document it somewhere our docker image have this script and the usage of this script.